analytics/utils/identify.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+const debug = require('debug')('analytics:identify');
+import Raven from 'shared/raven';
+import { amplitude } from './amplitude';
+
+export const identify = (userId: string, userProperties: Object) => {
+  const amplitudePromise = () => {
+    debug(`[Amplitude] Identify ${userId}`);
+    return amplitude.identify({
+      userId,
```

Comment:
Can we hash the `userId` at this central point instead of manually all over the place? That'd make it much easier to not forget to hash it, since it'll always be hashed when sent to amplitude.

Reply:
Done in https://github.com/withspectrum/spectrum/pull/3048/commits/c57122585de2c8f1696c0714be239268db4978bd